Framework/Built_In_Automation/Web/Selenium/BuiltInFunctions.py

```diff
@@ -789,6 +804,13 @@ def Go_To_Link(step_data, page_title=False):
                     # any other shared capabilities can be added from the selenium document
                     capabilities[left.strip()] = right.strip()
 
+            # Todo: profile, argument, extension, chrome option => go_to_link
+            elif mid.strip().lower() in ("chrome option", "chrome options") and dependency["Browser"].lower() == "chrome":
+                browser_options.append([left, right])
```

Comment:
append left.lower() and replace (" ", "_", "-") with ""

Reply:
this has been done on line 788. doing this again would be repetitive. did right.strip()